packages/ui/atomic/create-atomic/scripts/packageTemplate.json

```diff
@@ -9,15 +9,11 @@
   "collection": "dist/collection/collection-manifest.json",
   "collection:main": "dist/index.js",
   "scripts": {
-    "setup-lambda": "node ./scripts/setup-lambda.mjs",
     "setup-cleanup": "node ./scripts/clean-up.js",
-    "start": "node start-netlify.mjs",
+    "start": "stencil build --dev --watch --serve",
     "build": "stencil build",
-    "dev:live": "npm run site:link && node start-netlify.mjs --live",
-    "site:variables": "netlify env:import .env",
-    "site:init": "netlify init && npm run site:variables",
-    "site:deploy": "npm run site:link && npm run build && npm run site:variables && netlify deploy --prod --open",
-    "postinstall": "npm run setup-cleanup && prettier --write . --loglevel warn",
-    "prepare": "npm run setup-lambda"
+    "build:deploy": "node deploy.esbuild.mjs",
```

Comment:
The name `deploy` (in both the `package.json` script name and the file name), intuitively, makes me think that this script may also deploy something online.

I think we could use something like "deployable" instead.